src/main/target/BEESTORM/config.c

```diff
@@ -0,0 +1,156 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <platform.h>
+
+#ifdef USE_TARGET_CONFIG
+
+#include "common/axis.h"
+#include "common/maths.h"
+
+#include "config/feature.h"
+
+#include "drivers/light_led.h"
+#include "drivers/pwm_esc_detect.h"
+
+#include "fc/config.h"
+#include "fc/controlrate_profile.h"
+#include "fc/rc_modes.h"
+#include "fc/rc_controls.h"
+
+#include "flight/mixer.h"
+#include "flight/pid.h"
+
+#include "pg/vcd.h"
+
+#include "rx/rx.h"
+
+#include "io/serial.h"
+#include "io/osd.h"
+
+#include "sensors/battery.h"
+#include "sensors/gyro.h"
+
+#include "telemetry/telemetry.h"
+
+#define BEST_FRSKY_RSSI_CH_IDX     9
+
+#ifdef BRUSHED_MOTORS_PWM_RATE
+#undef BRUSHED_MOTORS_PWM_RATE
+#endif
+
+#define BRUSHED_MOTORS_PWM_RATE 32000           // 32kHz
+
+void targetConfiguration(void)
+{
+    if (hardwareMotorType == MOTOR_BRUSHED) {
+        motorConfigMutable()->dev.motorPwmRate = BRUSHED_MOTORS_PWM_RATE;
+        motorConfigMutable()->minthrottle = 1030;
+        pidConfigMutable()->pid_process_denom = 1;
+    }
+
+    for (uint8_t pidProfileIndex = 0; pidProfileIndex < MAX_PROFILE_COUNT; pidProfileIndex++) {
+        pidProfile_t *pidProfile = pidProfilesMutable(pidProfileIndex);
+
+        pidProfile->pid[PID_ROLL].P  = 86;
+        pidProfile->pid[PID_ROLL].I  = 50;
+        pidProfile->pid[PID_ROLL].D  = 60;
+        pidProfile->pid[PID_PITCH].P = 90;
+        pidProfile->pid[PID_PITCH].I = 55;
+        pidProfile->pid[PID_PITCH].D = 60;
+        pidProfile->pid[PID_YAW].P   = 123;
+        pidProfile->pid[PID_YAW].I   = 75;
+    }
+
+    for (uint8_t rateProfileIndex = 0; rateProfileIndex < CONTROL_RATE_PROFILE_COUNT; rateProfileIndex++) {
+        controlRateConfig_t *controlRateConfig = controlRateProfilesMutable(rateProfileIndex);
+
+        controlRateConfig->rcRates[FD_YAW] = 120;
+        controlRateConfig->rcExpo[FD_ROLL] = 15;
+        controlRateConfig->rcExpo[FD_PITCH] = 15;
+        controlRateConfig->rcExpo[FD_YAW]  = 15;
+        controlRateConfig->rates[FD_ROLL]  = 85;
+        controlRateConfig->rates[FD_PITCH] = 85;
+    }
+
+    batteryConfigMutable()->batteryCapacity = 250;
+    batteryConfigMutable()->vbatmincellvoltage = 28;
+    batteryConfigMutable()->vbatwarningcellvoltage = 33;
+
+    *customMotorMixerMutable(0) = (motorMixer_t){ 1.0f, -0.414178f,  1.0f, -1.0f };    // REAR_R
```

Comment:
What is this used for, and why can't your target just use QUAD_X?

Reply:
BeeStrom and BEEBRAIN_V2F is TinyWhoop style mount only.

https://newbeedrone.com/collections/flight-controller/products/beebrain-v2-micro-fpv-flight-controller